### DISCUSSION TITLE
Is it possible to use another solver instead of SCIP in ecole?

### DISCUSSION THREAD

**Discussion by dxyzx0** [ds4dm/ecole#247] [Questions] [answered]
I want to try a different B&B solver to test the performance.
Is there any possibility to use any other solvers instead of SCIP? 
What are the interfaces that the solver must provide?

**Comment by AntoinePrv** [comment, marked as answer]
Hello, @DingXiangYuanZhiXing 

It is not possible to use a solver different than SCIP. Making a common interface for different solvers is already a difficult project on its own.
